### PR TITLE
fix multiple creation of tooltip element for each chart in document, fix tooltip added to the correct document.body or document.fullscreenElement (depending on fullscreen mode)

### PR DESCRIPTION
**Pull request by dankrusi**
fix multiple creation of tooltip element for each chart in document
fix tooltip added to the correct document.body or document.fullscreenElement (depending on fullscreen mode) (see https://github.com/vega/vega-tooltip/issues/506)

### CODE REVIEW

File: src/Handler.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ export class Handler {
   constructor(options?: Options) {
     this.options = {...DEFAULT_OPTIONS, ...options};
     const elementId = this.options.id;
+    this.el = null!; // We create the tooltip element later, when we need it. This also has the advantage that if you have multiple charts, not multiple elements are created, as each chart will grab the existing one when its needed.
```
Comment (domoritz) [inline src/Handler.ts:34]: How about we allow el to be null in the types. Then you can remove the `!` here.

**Comment by domoritz** [comment]
Cleaned up in https://github.com/vega/vega-tooltip/pull/508. Thanks for the pull request.

**Comment by dankrusi** [comment]
Hi Domoritz, super many thanks for cleaning up and merging in. Sorry for the delay, I don't mind polishing merge requests, just takes me a while since I need to do these things in my freetime... Best, -dan

**Comment by domoritz** [comment]
Thanks for your help. Figuring out the issue and the fix is always the hardest part and you had already done that.

**Pull request closed without merging**